packages/core/src/enqueue_links/enqueue_links.ts

```diff
@@ -332,7 +354,7 @@ export async function enqueueLinks(options: SetRequired<EnqueueLinksOptions, 're
         }
 
         // Generate requests based on the user patterns first
-        const generatedRequestsFromUserFilters = createRequests(requestOptions, urlPatternObjects);
+        const generatedRequestsFromUserFilters = createRequests(requestOptions, urlPatternObjects, urlBlacklistPatternObjects);
```

Comment:
You'll need to apply this to the if code above too at line 353 (in the current code)

Reply:
Done!